platforms/unix/vm/sqUnixMain.c

```
@@ -1692,11 +1689,12 @@ SqModuleDefine(vm, Module);
 /*** options processing ***/
 
 
-static void usage(void)
+static void usage(int exitValue)
 {
   struct SqModule *m= 0;
   printf("Usage: %s [<option>...] [<imageName> [<argument>...]]\n", argVec[0]);
   printf("       %s [<option>...] -- [<argument>...]\n", argVec[0]);
+  printf("options begin with single -, but -- prefix is silently accepted\n");
```

The problem is that single-hyphen default is then what is advised by --help.  Why isn't it preferable to conform to Unix conventions [1] "The GNU double-hyphen option leader was chosen so that traditional single-letter options and GNU-style keyword options could be unambiguously mixed on the same command line. "  
[1] http://www.catb.org/~esr/writings/taoup/html/ch10s05.html

The single-hyphen could be silently accepted for backward compatibility. Maybe better would be a deprecated message, but perhaps that could adversely affect existing scripts(?).

How do Squeakers feel about that?

---

Based on limited information on the need for this, Eliot's reply "leave it as is" [1] .
Please revert and if there is pressing need, raise discussion on [vm-dev] list.  

[1] http://forum.world.st/Re-OpenSmalltalk-opensmalltalk-vm-command-line-arguments-use-single-dash-prefix-but-also-accept-doub-td4945805.html

---

See the explanation in the first comment. Unix apps allow multiple single letter options to merge into one word (e.g. grep -lir for grep -l -i -r), so GNU introduced an extra dash for word options. Multi-platform virtual machines like qemu-* or squeak don't allow collapse of single letter options, so an extra dash is superfluous. It does eat into max line length without adding any value.

I have no stakes either way. I just want to fix it once, skip bike-shedding and move onto other important stuff.